broker/broker.go

```diff
@@ -201,6 +201,15 @@ func (broker *ServiceBroker) addService(svc *service.Service) {
 // createService create a new service instance, from a struct or a schema :)
 func (broker *ServiceBroker) createService(svc interface{}) (*service.Service, error) {
 	schema, isSchema := svc.(moleculer.ServiceSchema)
+
```

Comment:
Hi @vpashka ,

Thank you for the interest in moleculer go and for you contribution :)

This is good.

My only recommendation is to move the whole logic to a separate function.
E.g
```
func (broker *ServiceBroker) resolveSchema(svc interface{}) (moleculer.ServiceSchema, bool) {
  schema, isSchema := svc.(moleculer.ServiceSchema)
  if !isSchema {
		s, ok := svc.(*moleculer.ServiceSchema)
		if ok {
			schema = *s
			isSchema = ok
		}
	}

  return schema, isSchema
}
```

Reply:
Done.. )